WcaOnRails/app/controllers/admin/avatars_controller.rb

```diff
@@ -20,7 +20,7 @@ def update_all
           when "defer"
             # do nothing!
           else
-            raise "Unrecognized avatar action '#{args[:action]}'"
+            fail "Unrecognized avatar action '#{args[:action]}'"
```

Comment:
Ok, good to know. Without a professional ruby background, it's hard for me distinguish between convention and "weird stuff". I often miss python with it's "one right way to do things" philosophy...

I'll change rubocop to require that we use `raise` everywhere.